pkg/controllers/metrics/pod/controller.go

```diff
@@ -68,42 +69,42 @@ var (
 	PodStartupDurationSeconds = opmetrics.NewPrometheusSummary(
 		crmetrics.Registry,
 		prometheus.SummaryOpts{
-			Namespace:  "karpenter",
+			Namespace:  metrics.Namespace,
 			Subsystem:  metrics.PodSubsystem,
 			Name:       "startup_duration_seconds",
-			Help:       "The time from pod creation until the pod is running.",
+			Help:       "The time from when Karpenter first thought the pod could schedule until it started running.",
 			Objectives: metrics.SummaryObjectives(),
 		},
 		[]string{},
 	)
 	PodBoundDurationSeconds = opmetrics.NewPrometheusHistogram(
```

Comment:
Should this be scheduled instead of bound? We haven't released this change yet so we still have a chance to change it -- I still personally prefer scheduled but I'm open to discussing

Reply:
Keeping it as bound just to reduce the number of changes, since we've creeped up a bit in this pr already